pkg/JuliaInterface/tst/wrapper.tst

```diff
@@ -14,6 +14,7 @@ gap> Julia.Base.typeof(N);
 <Julia: BigInt>
 gap> N(1);
 Error, MethodError: objects of type BigInt are not callable
+Maybe you forgot to use an operator such as *, ^, %, / etc. ?
```

Comment:
I don't see what the purpose of this test is supposed to be. It checks that a Julia `BigInt` is not callable, but how is that useful to use?
```suggestion
```